packages/asl-components/src/markdown/index.jsx

```diff
@@ -41,9 +42,10 @@ export default function Markdown({
         includeElementIndex={true}
         components={{
             ...(!links && components),
-            p: wrapInSpanIfOnlyChild(unwrapSingleLine, paragraphProps)
-        }}
-        remarkPlugins={significantLineBreaks ? [remarkBreaks] : []}
+            p: wrapInSpanIfOnlyChild(unwrapSingleLine, paragraphProps),
+            mark: ({ ...props }) => <mark style={{ backgroundColor: '#fff7bf', color: '#594d00', padding: '5px', fontWeight: 400 }} {...props} />
```

Comment:
I'd rather not have inline styles here.

```suggestion
            mark: ({ ...props }) => <mark className='govuk-tag govuk-tag--yellow' {...props} />
```

Note: the govuk-tag class may not work as is as we are not fully using the latest govuk css. If that class doesn't work, then use a custom class (e.g. `asl-highlight`) and add the neccessary css for that class here: https://github.com/UKHomeOffice/aspel-workspace/tree/main/packages/asl-components/styles

Note also the lint failure. I'd advise setting IntelliJ to run `eslint --fix` on save:
![Screenshot 2025-06-04 at 16 44 02](https://github.com/user-attachments/assets/40e0e1aa-a2f9-49e1-a44b-9b417c685357)